### ISSUE TITLE
Accessibility: The project name in header is enclosed in an h1 tag

### ISSUE THREAD

**Issue by nicolaisweb** [globaleaks/GlobaLeaks#3435]
### What version of GlobaLeaks are you using?

4.11.5

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### Describe the issue

The project name in header is enclosed in an h1 tag

### Proposed solution

In the home page the behavior is correct, since there is no title in the main content.

In the inner pages (the questionnaire first of all), the h1 should disappear from the project title (it can easily become a "span" tag) and should be put in the page title (or step) the user is browsing.

This implies that each page (or step) is provided with an h1 title.
